crates/rooch/src/commands/account/commands/update.rs

```diff
@@ -0,0 +1,136 @@
+    identifier::Identifier,
+    language_storage::{ModuleId, StructTag, TypeTag},
+};
+use moveos_types::module_binding::ModuleBundle;
+use moveos_types::{move_types::FunctionId, transaction::MoveAction};
+use once_cell::sync::Lazy;
+use rooch_framework::{
+    bindings::{
+        ecdsa_k1_recoverable_validator::EcdsaK1RecoverableValidator,
+        ecdsa_k1_validator::EcdsaK1Validator, ed25519_validator::Ed25519Validator,
+        schnorr_validator::SchnorrValidator,
+    },
+    ROOCH_FRAMEWORK_ADDRESS,
+};
+use rooch_key::keystore::AccountKeystore;
+use rooch_rpc_api::jsonrpc_types::ExecuteTransactionResponseView;
+use rooch_types::{
+    address::RoochAddress,
+    crypto::{BuiltinScheme, PublicKey},
+    error::{RoochError, RoochResult},
+};
+
+use crate::cli_types::{CommandAction, WalletContextOptions};
+use std::str::FromStr;
+
+/// Update an address with a new keypair from scheme to rooch.keystore
+#[derive(Debug, Parser)]
+pub struct UpdateCommand {
+    /// Rooch address in string format.
+    #[clap(short = 'a', long = "address")]
+    address: String,
+    #[clap(short = 'm', long = "mnemonic-phrase")]
+    mnemonic_phrase: String,
+    #[clap(flatten)]
+    pub context_options: WalletContextOptions,
+    /// Command line input of crypto schemes (ed25519, multied25519, ecdsa, or schnorr)
+    #[clap(short = 's', long = "scheme", arg_enum)]
+    pub crypto_schemes: BuiltinScheme,
+}
+
+#[async_trait]
+impl CommandAction<ExecuteTransactionResponseView> for UpdateCommand {
+    async fn execute(self) -> RoochResult<ExecuteTransactionResponseView> {
+        println!("{:?}", self.mnemonic_phrase);
+
+        let mut context = self.context_options.build().await?;
+
+        match BuiltinScheme::from_flag_byte(self.crypto_schemes.flag()) {
+            Ok(scheme) => {
+                let existing_address =
+                    RoochAddress::from_str(self.address.as_str()).map_err(|e| {
+                        RoochError::CommandArgumentError(format!(
+                            "Invalid Rooch address String: {}",
+                            e
+                        ))
+                    })?;
+
+                let public_key: PublicKey = context
+                    .config
+                    .keystore
+                    .update_address_with_key_pair_from_scheme(
+                        &existing_address,
+                        self.mnemonic_phrase,
+                        scheme,
+                        None,
+                    )
+                    .map_err(|e| RoochError::UpdateAccountError(e.to_string()))?;
+
+                println!(
+                    "{}",
+                    AccountAddress::from(existing_address).to_hex_literal()
+                );
+                println!(
+                    "Generated a new keypair for an existing address on scheme {:?} [{existing_address}]",
+                    scheme.to_owned()
+                );
+
+                let address = match scheme {
```

Comment:
I've updated it. However, due to cyclic import dependencies between `rooch-types` and `rooch-framework`, I couldn't import Validators' address and module into BuiltinScheme.